modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/css/components/FieldSelectModalContent.scss

```diff
@@ -0,0 +1,35 @@
+@import 'atlas-variables';
+@import 'cadmin-variables';
+
+html#{$cadmin-selector} .cadmin .field-select-modal {
+	.selection-mode-multiple {
+		.custom-control-label {
+			line-height: 24px;
+		}
+
+		.field-label {
+			margin-left: 4px;
+		}
+
+		.treeview-group .autofit-col:has(> .custom-checkbox) {
+			width: 100%;
+		}
+
+		.treeview-link .custom-checkbox label {
+			display: block;
+			width: 100%;
+		}
```

Comment:
This ensures that click anywhere in row toggles the checkbox, by expanding label to full width.